apps/elewa-group-website/src/_variables.scss

```diff
@@ -56,4 +56,8 @@
   --elewa-group-website-font-size-nav-link: small;
   --elewa-group-website-font-size-nav-link-medium: medium;
 
+  //jobs-section
+  --elewa-group-website-font-size-paragraph: 40px;
+  --elewa-group-website-font-size-paragraph-medium: 27px;
+
```

Comment:
component styles for non-reusable components can be placed in their scss files, no need to put them here, use rem for font size instead of px

Reply:
I have resolved this issue